PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Application/Responses/ServiceResponse.cs

```diff
@@ -0,0 +1,26 @@
+namespace PrivateHospitals.Application.Responses;
+
+public class ServiceResponse<T>
```

Comment:
It's usually called `Result<T>` and it's called result pattern. It's ok to make your way, but usually we name it Result

Reply:
Fixed